engine/menu/debug_menu.asm

```diff
@@ -82,26 +82,151 @@ DebugMenuOptionSubGames::
 DebugMenuOptionMonsterTest::
 	ld hl, wPokedexOwned
 	ld de, wPokedexSeen
-	ld b, $1F
-	ld a, $FF
+	ld b, 31
+	ld a, -1
 .loop
 	ld [hl+], a
```

Comment:
`ld [hli], a`; the pret projects don't use `[hl+]`/`[hl-]`, nor `ldi [hl], a`/`ldd [hl], a`